### PR TITLE
(Feature) Orderbook current list styling

### LINKED ISSUE #20: Revisit Orderbook (current) list styling

### PR DESCRIPTION
**Pull request by gabitoesmiapodo**
Closes #20 

It should look better now.

screen shot 2019-02-20 at 16 37 36
screen shot 2019-02-20 at 16 37 55
screen shot 2019-02-20 at 16 38 07

### CODE REVIEW

File: src/components/common/empty_content.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React, { HTMLAttributes } from 'react';
+import styled from 'styled-components';
+
+interface Props extends HTMLAttributes<HTMLDivElement> {
+    alignAbsoluteCenter?: boolean;
```
Comment (fvictorio) [inline src/components/common/empty_content.tsx]: The two usages of this component set this to `true`. Are we anticipating a component that needs to use this with `false`?

File: src/components/common/empty_content.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    text?: string;
```
Comment (fvictorio) [inline src/components/common/empty_content.tsx]: Can we make this prop mandatory?

File: src/components/common/empty_content.tsx (hunk 1 above)
Comment (gabitoesmiapodo) [reply src/components/common/empty_content.tsx]: Yes, you might not need (or not want to) to center it occupying all of its container, or you may not have a parent with a `position: relative;` property (which could be disastrous).

File: src/components/common/empty_content.tsx (hunk 2 above)
Comment (gabitoesmiapodo) [reply src/components/common/empty_content.tsx]: `text?: string;` is mandatory now.

**Pull request merged**